contrib/meson.build

```diff
@@ -3,6 +3,12 @@ if get_option('qubes')
   subdir('qubes')
 endif
 
+install_data([
```

Comment:
If it's installing by default it's not really contrib/ I'd argue.

Reply:
Do you think it's a useful script? I'm thinking about the IOT case where maybe you want to pregenerate the database at build time (and take a 7MB hit on disk space) to avoid a few second initial start delay. Maybe we just drop it?

Reply:
It's useful, I just don't think we should be storing it in contrib by default if it's installed independently to `/usr/share/fwupd`.

Can we put it in generate-build?

Reply:
Or another idea is that we have a fwupdtool command that does the same thing.  Then in the rpm/deb postinst we call `fwupdtool generate-quirk-db` or similar.

Reply:
Actually, `sudo fwupdtool get-plugins` should generate the quirk db automatically -- I just figured a python script was more polite to run in a distro build script.

Reply:
Lets leave it out for now, and add it back when/if it's needed. Maybe I just need to make the initial db creation faster!